### PR TITLE
More GUI integration tests

### PR DESCRIPTION
**Pull request by gz-c**
- /address_uxouts
- /block
- /last_blocks
- /network/connections
- /network/connection
- /network/defaultConnections
- /network/connections/trust
- /network/connections/exchange

Also cleanup test-fixtures filenames (do not use mixed case filenames).
Fix bug with `/network/connections` when networking is disabled (pool's strand loop was not run).
Fix bug with `SendOr404` for `/network/connection` (see #1021, requires total removal of these methods).

### CODE REVIEW

**Comment by iketheadore** [comment]
Init TEST="" in integration-test-live.sh, otherwise the live test will return "TEST unbound variable" error

File: src/daemon/gnet/pool.go [hunk 1]
```diff
@@ -263,6 +254,26 @@ loop:
 	return nil
 }
 
+// RunOffline runs the pool in offline mode. No connections will be accepted,
+// but strand requests are processed.
+func (pool *ConnectionPool) RunOffline() error {
+	pool.processStrand()
+	return nil
+}
```
[the remark this answers is not in the document]
Comment (gz-c) [reply src/daemon/gnet/pool.go]: Matches API for Run()

File: src/api/cli/integration/integration_test.go [hunk 2]
```diff
@@ -70,21 +70,21 @@ func TestGenerateAddresses(t *testing.T) {
 			[]string{"generateAddresses"},
 			[]byte("7g3M372kxwNwwQEAmrronu4anXTW8aD1XC\n"),
 			filepath.Join(walletDir, walletName),
-			filepath.Join("testdata", "generateAddresses.golden"),
+			filepath.Join("test-fixtures", "generate-addresses.golden"),
```
[the remark this answers is not in the document]
Comment (gz-c) [reply src/api/cli/integration/integration_test.go]: Refactoring cli and gui integration tests to use the same update method for golden files and to use the directory constants

**Comment by gz-c** [comment]
fixed

**Pull request merged**